Source/Bugsnag.h

```diff
@@ -183,6 +183,33 @@ static NSString *_Nonnull const BugsnagSeverityInfo = @"info";
  */
 + (void)leaveBreadcrumbForNotificationName:(NSString *_Nonnull)notificationName;
```

Comment:
It may be outside the scope of this PR, but it'd be good to remove any non-compliant methods for leaving breadcrumbs such as these from our public interface. This could be done either by removing them entirely, or just removing them from the public header.

Reply:
This will be handled as a separate piece of work.